boards/arm/degu_evk/doc/index.rst

```diff
@@ -0,0 +1,20 @@
+.. _degu_evk:
+
+Degu Evaluation Kit
+####################
+
+Overview
+********
+
```

Comment:
Sparse documentation indeed, but I'll yield to @carlescufi comment.

Still, let's rewrite what's here as (and link to their English website):
```
The Degu Evaluation Kit is an IoT device for sensors and actuators
and can connect to an OpenThread mesh network.  The Kit has a
Nordic nRF52840 SoC and NXP A71CH Secure Element, and can
connect some Seeed Grove sensors via GPIO, ADC, I2C,
and UART.

.. figure:: img/degu_evk.png

For more information, see the `Degu Project website <https://open-degu.com/en/>`__.
```

Reply:
Thanks you!
I fixed it.